index.bs

```diff
@@ -2244,7 +2253,7 @@ Base64-encoded string.
       }
 
       browsingContext.BoxClipRectangle = {
-         type: "viewport",
+         type: "box",
```

Comment:
FYI I also already commented on the wpt test PR but want to repeat here that this is a backward incompatibility change and we should at least coordinate a release for both Chrome and Firefox.

Reply:
Answering here: https://github.com/web-platform-tests/wpt/pull/42524#discussion_r1360360344